presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -668,4 +673,13 @@ public static Object deserializePartitionValue(Type type, String valueString, St
         // Iceberg tables don't partition by non-primitive-type columns.
         throw new PrestoException(GENERIC_INTERNAL_ERROR, "Invalid partition type " + type.toString());
     }
+
+    public static void loadCachingProperties(Map<String, String> properties, IcebergConfig icebergConfig)
+    {
+        properties.put(IO_MANIFEST_CACHE_ENABLED, "true");
+        properties.put(FILE_IO_IMPL, String.valueOf(icebergConfig.getFileIOImpl()));
```

Comment:
minor: I would probably still pull this out of `loadCachingProperties` as this makes readers of the code think that this is related to caching (which it isn't)

Reply:
@nastra As of now I have kept it to set for `loadCachingProperties`, since setting `io-impl` is required for initialization of FileIO to set catalog properties on iceberg side. And if manifest file caching is not enabled we don't have to set this explicitly.